config/initializers/stripe.rb

```diff
@@ -4,6 +4,8 @@
 # a bit cleaner than accessing keys in different ways.
 module Stripe
   class << self
+    # Returns the value of Stripe.publishable_key and Stripe.endpoint_secret.
+    # Attribute values can also be set by doing Stripe.publishable_key = <your_new_value>
```

Comment:
I think it would be worth also to mention `Stripe.endpoint_secret = <your_new_value>`

Reply:
Would you like to address this comment @as1729? Then it can be approved and merged